Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -667,11 +792,12 @@ def perform_rasterize(path: str | list[str],
                           f" {MAX_RASTERIZATIONS_COUNT=}, {len(browser.list_tab())=}")
             if total_rasterizations_count > MAX_RASTERIZATIONS_COUNT:
                 demisto.info(f"Terminating Chrome after {total_rasterizations_count} rasterizations")
-                terminate_chrome(browser)
+                terminate_chrome(killall=True)
+                write_file(CHROME_INSTANCES_FILE_PATH, "", overwrite=True)
                 demisto.info(f"Terminated Chrome after {total_rasterizations_count} rasterizations")
-                write_info_file(RASTERIZATIONS_COUNTER_FILE_PATH, "0")
+                write_file(RASTERIZATIONS_COUNTER_FILE_PATH, "0", overwrite=True)
```

Comment:
Where do we call this method with overwrite set to false?

Reply:
when we are initiating new chrome instance, we dont want to overwrite the file and just append a new row.
inside start_chrome_headless function:
write_file(CHROME_INSTANCES_FILE_PATH, new_row)